hs-tendermint/hs-tendermint.cabal

```diff
@@ -0,0 +1,49 @@
+cabal-version: 1.12
```

Comment:
how did this get in here? I think you must have pulled in iraklis work?

Reply:
Yeah not sure how I got this. I'll cherry pick out the commits